examples/schnorr.c

```diff
@@ -140,9 +140,8 @@ int main(void) {
      * example through "out of bounds" array access (see Heartbleed), Or the OS
      * swapping them to disk. Hence, we overwrite the secret key buffer with zeros.
      *
-     * TODO: Prevent these writes from being optimized out, as any good compiler
+     * Here we are preventing these writes from being optimized out, as any good compiler
      * will remove any writes that aren't used. */
-    memset(seckey, 0, sizeof(seckey));
-
+    secure_erase(seckey,sizeof(seckey));
```

Comment:
```suggestion
    secure_erase(seckey, sizeof(seckey));
```

And the same for the other calls to `secure_erase`.

Reply:
> That was really the last comment I have.

Oh that's totally fine. Even I am sceptical about the coding standards for established projects :) 
Done. Thanks for suggesting!!